packages/client/hmi-client/src/temp/tera-simulation-workflow.vue

```diff
@@ -1,22 +1,86 @@
 <template>
-	<infinite-canvas debug-mode :last-transform="lastTransform" @save-transform="saveTransform">
+	<infinite-canvas
+		debug-mode
+		@contextmenu="toggleContextMenu"
+		:last-transform="canvasTransform"
+		@save-transform="saveTransform"
+	>
 		<template #foreground></template>
 		<template #data>
-			<div style="font-size: 24px; padding: 10px; background: #9ef">This is a DataLayer DIV</div>
+			<ContextMenu ref="contextMenu" :model="contextMenuItems" />
+
+			<ul v-for="node in nodes">
+				<tera-workflow-node :node="node" :initialTransform="canvasTransform"></tera-workflow-node>
```

Comment:
Is this related to what we were discussing yesterday about needing transforms x/y/k? 
I don't see initialTransform getting used or declared within tera-workflow-node.